pci/src/msix.rs

```diff
@@ -72,6 +72,8 @@ struct MsixConfigState {
 pub struct MsixConfig {
     pub table_entries: Vec<MsixTableEntry>,
     pub pba_entries: Vec<u64>,
+    #[cfg(target_arch = "aarch64")]
+    pub devid: u32,
```

Comment:
To discuss: Different from X86, Arm need to set `devid `in `kvm_irq_routing_msi`, so I added this field to save the devid (BDF) created in device_manager.rs and finally pass it to interrupt.rs to update GSI routing entry. 
Problem is, this file and other modified files in pci/ folder are not architecture specific, but now I have to add `#[cfg(target_arch = "aarch64")]` here and there to distinguish. 
Can you suggest a better way?

Reply:
I think it's fine to not make it conditional. If we safe the BDF for all architectures it may also be helpful for future debugging.